platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheEncryptionIntegrationTest.groovy

```diff
@@ -71,6 +72,7 @@ class ConfigurationCacheEncryptionIntegrationTest extends AbstractConfigurationC
     }
 
     def "configuration cache encryption enablement is #enabled if kind=#kind"() {
+        Assume.assumeTrue("Compression no longer exposes the raw data", enabled)
```

Comment:
🤔 I'm not sure what to do with this test. @abstratt, do you think we still need it?

Reply:
One thing we are losing here is the ability of detecting a regression where a bug introduced to CC effectively disabled encryption.     

We would need to make compression optional (even if behind an internal option), so we could disable it for this test case. May be a good thing to have (for debugging compression performance in the field), and seems more or less trivial to implement, but I will not push for it.